File: tests/Generator/resource/python/complex/security_o_auth2.py
from pydantic import BaseModel, Field, GetCoreSchemaHandler
from pydantic_core import CoreSchema, core_schema
from typing import Any, Dict, Generic, List, Optional, TypeVar, UserList, UserDict
from .security import Security


class SecurityOAuth(Security):
    token_url: Optional[str] = Field(default=None, alias="tokenUrl")
    authorization_url: Optional[str] = Field(default=None, alias="authorizationUrl")
    scopes: Optional[List[str]] = Field(default=None, alias="scopes")
    pass

File: tests/Generator/resource/swift/complex/SecurityOAuth2.swift
class SecurityOAuth: Security {
    var tokenUrl: String
    var authorizationUrl: String
    var scopes: Array<String>

    enum CodingKeys: String, CodingKey {
        case tokenUrl = "tokenUrl"
        case authorizationUrl = "authorizationUrl"
        case scopes = "scopes"
    }
}

